src/pages/Login.vue

```diff
@@ -222,7 +221,6 @@ export default {
       otpInput: "", // otp input text
       resendOTPTimer: "", // the count of the timer

```

Comment:
the default value for this is still a string but is meant to old integer values.

Reply:
sorry i thought i changed it